app/Http/Controllers/AuthController.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace AtlasVG\Http\Controllers;
+
+use AtlasVG\Http\Controllers\Controller;
+use AtlasVG\TokenStore\TokenCache;
+use Illuminate\Http\Request;
+use Microsoft\Graph\Graph;
+use Microsoft\Graph\Model;
+
+class AuthController extends Controller {
+    public function signin() {
+        
+        $oauthClient = new \League\OAuth2\Client\Provider\GenericProvider([
```

Comment:
Maybe using service providers will improve the usage of the lib: **https://laravel.com/docs/5.8/providers**

Reply:
good idea -> future enhancements